### PR TITLE
Add Site commands to SDK

### PR DESCRIPTION
**Pull request by tfevens**
This adds the Site Commands endpoints into the SDK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -348,6 +348,14 @@ On a `Webhook` instance you may also call:
 $webhook->delete();
 ```
 
+### Site Commands
+
+```php
+$forge->createSiteCommand($serverId, $siteId, array $data);
```
Comment (jbrooksuk) [inline README.md:354]: I would call this `executeSiteCommand`.

File: src/MakesHttpRequests.php [hunk 2]
```diff
@@ -72,7 +72,9 @@ protected function request($verb, $uri, array $payload = [])
             empty($payload) ? [] : ['form_params' => $payload]
         );
 
-        if ($response->getStatusCode() != 200) {
+        $statusCode = $response->getStatusCode();
+
+        if ($statusCode < 200 || $statusCode > 299) {
```
Comment (jbrooksuk) [inline src/MakesHttpRequests.php:77]: This change seems unrelated?

**Review by jbrooksuk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tfevens) [reply README.md]: Changed.  I think I had written this up prior to the change from create to execute, so I had the old naming.

File: src/MakesHttpRequests.php (hunk 2 above)
Comment (tfevens) [reply src/MakesHttpRequests.php:77]: This was done as the Execute Command call returns a 204 code on success.  Prior to the change, the code was looking for ONLY a 200 response, therefore successful calls were returning an exception (from the handleRequestError() method).

This change allows for any 2xx responses to return successfully, with the response body handed back to the requestor.

Comment (jbrooksuk) [reply src/MakesHttpRequests.php:77]: Gotcha, makes sense.

**Review by jbrooksuk — APPROVED** [review]

**Comment by driesvints** [comment]
Thanks!
